src/app/js/lib/components/searchbar/SearchBar.js

```diff
@@ -45,7 +40,7 @@ const SearchBar = ({
     value,
     onChange,
     onClear,
-    actions,
```

Comment:
It seems that SearchBar actions for toggle facets are used in DatasetSearchBar in graph pages